install/kubernetes/cilium/values.yaml

```
@@ -188,6 +188,10 @@ bpf:
   # memory usage but can reduce latency.
   preallocateMaps: false
 
+  # socketLBHostnsOnly disables socket lb for non-root ns. This is used to enable
+  # Istio routing rules.
+  socketLBHostnsOnly: false
```

FWIW this won't show up in the Helm values documentation correctly unless it begins with a `--`:

```suggestion
  # -- Disable socket lb for non-root ns. This is used to enable Istio routing rules.
  socketLBHostnsOnly: false
```

PR #14083 will fix up a bunch of the other cases, I think you'll need to rebase when that goes in anyway but it would be good to get the documentation generation working correctly here when that happens.

---

Done.